example.py

```diff
@@ -8,8 +8,14 @@
 
 app = Flask("foobar")
 
+@app.template_filter()
+def is_list(value):
+    return isinstance(value, list)
+
+app.jinja_env.filters['is_list'] = is_list
+
 url = os.environ.get("URL", "localhost:5000")
-registry = SwaggerApiRegistry(app, baseurl="http://{}/api/v1".format(url))
+registry = SwaggerApiRegistry(app, baseurl="http://localhost:5000/api/v1".format(url))
```

Comment:
Good catch, `url = os.environ.get("URL", "localhost:5000")` should take care of this